modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/implictis/BatchAddressInfo.scala

```diff
@@ -0,0 +1,35 @@
+package org.ergoplatform.explorer.http.api.v1.implictis
+
+import org.ergoplatform.explorer.Address
+import org.ergoplatform.explorer.http.api.v1.models.AddressInfo
+import io.circe._
+import io.circe.generic.semiauto._
+import io.circe.syntax._
+import org.ergoplatform.explorer.http.api.v1.implictis.schemaForMapKV.schemaForMap
+import sttp.tapir.{Schema, Validator}
+import cats.instances.list._
+import cats.instances.either._
+import cats.syntax.traverse._
+
+object BatchAddressInfo {
+
+  implicit val AddressKeyEncoder: KeyEncoder[Address]   = (addr: Address) => addr.unwrapped
+  implicit val AddressInfoEncoder: Encoder[AddressInfo] = deriveEncoder[AddressInfo]
```

Comment:
Instances should be declared in type's companion objects when possible. No need to make orphan instances when companions are accessible in the codebase.